docs/resources/database_role.md

```diff
@@ -5,20 +5,25 @@ description: |-
   
 ---
 
+!> **V1 release candidate** This resource was reworked and is a release candidate for the V1. We do not expect significant changes in it before the V1. We will welcome any feedback and adjust the resource if needed. Any errors reported will be resolved with a higher priority. We encourage checking this resource out before the V1 release.
```

Comment:
what about such a comment in the deatasource docs?

Reply:
Added